regression/scripts/optimization_packages/optimization_packages.py

```diff
@@ -127,8 +127,8 @@ def setup(solver_name):
 
     problem.inputs = np.array([
     #   [ tag   , initial,(   lb   ,   ub   )     , scaling , units ]
-        [ 'x1'  ,  1.  , (   -2.   ,   2.   )  ,   1.   , 1*Units.less],
-        [ 'x2'  ,  1.  , (   -2.   ,   2.   )  ,   1.   , 1*Units.less],
+        [ 'x1'  ,  1.  ,    -2.   ,   2.     ,   1.   , 1*Units.less],
```

Comment:
I assume this is a stylistic change throughout the code.If so , we should update the tutorials as well and all the other instances of optimizations in the regressions

Reply:
We need to wait to update the tutorials until 2.4 is released. The regressions are all updated.

This change was necessary to allow problems to be hard bounded.